payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/NextActionSpec.kt

```diff
@@ -0,0 +1,196 @@
+package com.stripe.android.ui.core.elements
```

Comment:
This class is the meat of what has been added. It contains the data structures received from the server, and converting them to our internal representation